media.go

```diff
@@ -34,6 +34,30 @@ type InputMedia struct {
 	HasSpoiler           bool     `json:"has_spoiler,omitempty"`
 }
 
+type InputPaidMedia struct {
```

Comment:
`InputMedia` has the same set of fields, so we can avoid creating another structure